rails.md

```diff
@@ -5,29 +5,41 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. MVC (Model View Controller) is a pattern for the architecture of a software program. Give a brief description of each component and describe how Ruby on Rails handles MVC.
 
-  Your answer:
+  Your answer: 
+  
+  Model: It is how the data is represented. 
+  
+  View: It is how the user see the data presented. 
+  
+  Controller: Requests the data for the model
 
-  Researched answer:
+  Researched answer: 
+  
+  The Model component corresponds to all the data-related logic that the user works with.
+  
+  The View component is used for all the UI logic of the application.
+  
+  Controllers act as an interface between Model and View components to process all the business logic and incoming requests.
 
 
 
 2. Using the information given, fill in the blanks to complete the steps for creating a new view in a Rails application.
 
-  Step 1: Create the _________ in the file config/routes
+  Step 1: Create the route in the file config/routes
   ```
   get '/about' => 'statics#about'
   ```
 
-  Step 2: Create the ____________ in the file _____________
+  Step 2: Create the controller in the file app/controller 
   ```
-  class ____________ < ApplicationController
-    def ___________
-      render: ________________
+  class newController < ApplicationController
+    def controllerName
+      render: controllerName.html.erb
     end
   end
   ```
 
-  Step 3: Create the View in the file ______________
+  Step 3: Create the View in the file app/views/viewsfile
   code:
   ```
```

Comment:
Step 3: Create the View in the file app/views/statics/about.html.erb
Same thing here, we know the name of the view file if we did generated the controller